httperrors/httperrors_test.go

```diff
@@ -117,3 +117,15 @@ func TestNewUnauthorizedError(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, http.StatusText(http.StatusUnauthorized), dto.Status)
 }
+
+func TestNewBadRequestError(t *testing.T) {
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.